beets/mediafile.py

```diff
@@ -582,7 +582,10 @@ def set(self, mutagen_file, value):
         """Set an individual value as the only value for the field using
         this style.
         """
-        self.set_list(mutagen_file, [value])
+        if value is None:
```

Comment:
In fact, it's ok. `get` is implemented as:
```
          try:
              return self.get_list(mutagen_file)[0]
          except IndexError:
              return None
```
So None is returned when get_list returns an empty lists, that is, when the tag doesn't exist in the file. This allows to identify when a tag doesn't exist and allows to remove a tag from a file. Do you think that it would be better to return an empty string and always interpret an empty value as a non-existant tag? (so setting ... say artist to '' would delete the tag entry)